### PR TITLE
Tests assessment pagination

### PR DESCRIPTION
**Pull request by Vyacheslau**
Tests for pagination on assessment widget within Audit object.

### CODE REVIEW

File: test/selenium/src/lib/service/rest/client.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+from lib.constants import url
+from lib.utils.test_utils import append_random_string
+
+
+class RestClient(object):
+  """The class used for HTTP interactions with App's REST API"""
+  BASIC_HEADERS = {'X-Requested-By': 'gGRC',
+                   'Content-Type': 'application/json',
+                   'Accept-Encoding': 'gzip, deflate'}
+  AUTH = url.API + "/roles?name__in=Auditor&_=1463991908086"
```
Comment (alieh-rymasheuski) [inline test/selenium/src/lib/service/rest/client.py]: Hardcoded session id (or whatever `_` is).

File: test/selenium/src/lib/service/rest/client.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                                             title_postfix=title_postfix,
+                                             **kwargs)
+    headers = self.get_headers()
+    response = requests.post(url=self.url, data=request_body, headers=headers)
+    return response
+
+
+def generate_body_by_template(count, template_name, title_postfix=None,
+                              **kwargs):
+  """The function generates list of objects based on object type  (assessment,
```
Comment (alieh-rymasheuski) [inline test/selenium/src/lib/service/rest/client.py]: Please format this docstring as header - blank line - long description (or just reword it to fit in a single line).

Also, note that it would be better style to document it as `"""Generate list of objects ..."""` instead of `"""The function generates list of objects ..."""` (see [this section of pep8](https://www.python.org/dev/peps/pep-0008/#documentation-strings)).

File: test/selenium/src/lib/service/rest/client.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  control, etc) from json templates"""
+
+  def upgrade_template(template_name, title_postfix=None, **kwargs):
+    """The function return updated template json with random title by template
```
Comment (alieh-rymasheuski) [inline test/selenium/src/lib/service/rest/client.py]: Same docstring issue here.

File: test/selenium/src/lib/base.py [hunk 4]
```diff
@@ -406,6 +413,68 @@ def clear(self):
...
+            self._driver, PaginationControl.NEXT_PAGE_BUTTON),
+        self.LAST_PAGE: selenium_utils.get_when_visible(
+            self._driver, PaginationControl.LAST_PAGE_BUTTON)
+    }
+
+  def select_items_per_page(self, items_count):
+    """Selects items count to show per page"""
+    self.items_per_page_drop_down_toggle.click()
+    self.items_per_page_buttons[items_count].click()
+    time.sleep(3)  # TODO add waiter implementation
```
Comment (egorhm) [inline test/selenium/src/lib/base.py]: we need to get rid of this TODO

**Comment by alieh-rymasheuski** [comment]
Looks almost OK overall. Please fix the commented issues (the docstring issues are one-minute work anyway) and I'll approve the PR.

File: test/selenium/src/lib/constants/url.py [hunk 5]
```diff
@@ -3,11 +3,17 @@
 # Created By: jernej@reciprocitylabs.com
 # Maintained By: jernej@reciprocitylabs.com
 
+# pylint: disable=too-few-public-methods
+
+"""App enpoints constants"""
```
Comment (egorhm) [inline test/selenium/src/lib/constants/url.py]: should be `endpoints`
